splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -106,6 +107,14 @@ private void configureIngress() throws Exception {
             .end();
     }
 
+    private void configureSuccessHandler() throws Exception {
+        // If Event was sent successfully, send success reply to return kafka
+        from(direct("success"))
+            .setBody(simple("Response ${body}"))
+            .setHeader("outcome-success", simple("true"))
+            .to(kafka(kafkaReturnTopic).brokers(kafkaBrokers));
```

Comment:
Observation: This route is repeated second time (once for error message, and second here for success). Can we create a new direct route to avoid repeating?